lib/modules/manager/mix/artifacts.ts

```diff
@@ -75,26 +78,50 @@ export async function updateArtifacts({
 
   for (const { packageName } of updatedDeps) {
     if (packageName) {
-      const [, organization] = packageName.split(':');
+      const [, organization, ,] = packageName.split(':');
```

Comment:
```suggestion
      const [, organization] = packageName.split(':');
```
not needed

Reply:
I don't understand the suggestion. This wants to extract the second value in whatever is returned. The packageName when split can be 2 or 3 elements in length. The suggestion only works when array of length 2 is returned.

Reply:
no, the split returns all elements, but the array destruction will only return the first two and ignore the rest.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment